docs/source/cli-docs.md

```diff
@@ -44,10 +44,10 @@ kedro viz run [OPTIONS]
   - Whether to open the Kedro Viz interface in the default browser. The browser will open if the host is `localhost`. Defaults to `True`.
 
 - `--load-file <path>`
-  - Path to load Kedro Viz data from a directory. If provided, Kedro Viz will load the visualisation data from this path instead of generating it from the pipeline.
+  - Path to load Kedro Viz data from a [directory](https://docs.kedro.org/projects/kedro-viz/en/latest/cli-docs.html#kedro-viz-directory-structure-when-you-save-it-as-a-file). If provided, Kedro Viz will load the visualisation data from this path instead of generating it from the pipeline
```

Comment:
These absolute links won't work, and also they introduce an inconsistency (they use `/en/latest` but maybe the user was browsing `/en/stable` or `/en/10.1.0`), we should use relative links